platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpoint.java

```diff
@@ -212,23 +212,17 @@ private File createFileFromAttachement(Attachment attachment, Response response)
                     FileUtils.copyInputStreamToFile(inputStream, newFile);
 
                 } catch (IOException e) {
-                    logger.warn("Unable to write file.", e);
-                    Response.ResponseBuilder responseBuilder = Response.serverError();
-                    response = responseBuilder.build();
+                    LOGGER.warn("Unable to write file.", e);
```

Comment:
If the copy fails, shouldn't we set `newFile` to `null` so the calling method knows something didn't work? Or we could throw an exception like you suggested above.